app/workers/abstract_xikolo_course_worker.rb

```diff
@@ -50,7 +50,9 @@ def handle_response_data(response_data)
       course.description = convert_to_absolute_urls(parse_markdown(course_element['description']))
       course.course_instructors = course_element['lecturer']
       course.open_for_registration = !course_element['locked']
-      track = CourseTrack.find_by(course_id: course.id, track_type: course_track_type) || CourseTrack.create!(track_type: course_track_type, costs: 0.0, costs_currency: "\xe2\x82\xac")
+      track = CourseTrack.find_by(course_id: course.id, track_type: course_track_type) || CourseTrack.create!(track_type: course_track_type)
+      track.costs = 0.0
+      track.costs_currency = "\xe2\x82\xac"
```

Comment:
To be honest: I don't remember exactly. I know, that @sebastiankliem had a nasty bug  with the currency sign (instead of this notation), but I was not able to reproduce any (loading courses works, displaying these works as well, ...). Therefore, I replaced all occurrences with `€`.